estate/models/estate_property.py

```diff
@@ -1,6 +1,7 @@
-from odoo import fields, models
+from odoo import fields
+from odoo import models
 
-class estate_property(models.Model):
+class Estate_Property(models.Model):
```

Comment:
Pas de underscore en PascalCase
```suggestion
class EstateProperty(models.Model):
```